tests/test_compilesim.py

```diff
@@ -977,6 +1013,19 @@ def test_rom_out_of_range_error(self):
         with self.assertRaises(pyrtl.PyrtlError) as ex:
             self.sim(tracer=sim_trace)
 
+    def test_rom_romblock_in_memory_value_map_error(self):
+        rom_data_array = [1, 2, 3, 4, 5, 6, 7, 8]
```

Comment:
I don't think the value of `rom_data_array` or `mem_val_map` matter for this test? If that's right, consider simplifying them (reduce addrwidth to 1?), so it's more obvious to the reader that these values are not important